### ISSUE TITLE
Usage examples confusion

### ISSUE THREAD

**Issue by teemow** [giantswarm/cli#390]
Examples for `swarm up --help` look like this:

```
  -d                                    Create service using swarm.json in the current directory. Use -d to detach
  -d <filepath>                         Create and start service using the given config file
  -d [<filepath>] --var-file=<filepath> Create and start service using var-file. Variables grouped by env's in var-file, that defaults to swarmvars.json
  -d [<filepath>] --var=<key=value>     Create and start service using variables. The given key, used as $key in swarm.json, will be replaced with the given value
```

So you could assume that the parameter for the filepath is `-d <filepath>` but this is detach + filepath. Since filepath is a positional argument.

**Comment by teemow** [comment]
ping @marians

**Comment by marians** [comment]
Resolved via #392
